_posts/en/posts/2018-07-29-release-0.16.2.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Bitcoin Core 0.16.2 Released
+name: blog-release-0.16.2
+id: en-blog-release-0.16.2
+lang: en
+type: posts
+layout: post
+share: true
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+excerpt: >
+  Bitcoin Core 0.16.2 is now available with new bug fixes and minor
+  updates.
+---
+Bitcoin Core version 0.16.2 is now available for [download][download
+page].  All users are encouraged to upgrade to this [maintenance
+release][] that fixes several bugs and provides backports of new minor
+features, such as:
+
+- The [verifytxoutproof RPC][rpc verifytxoutproof] is no longer
+  vulnerable to a particular [expensive attack][tx-as-internal-node]
+  against SPV proofs publicly disclosed in early June.  The attack was
+  considered unlikely given that much cheaper attacks of roughly equal
+  effectiveness are well known.  Similarly, the [getblock RPC][rpc
+  getblock] also now returns extra information that can be used to
+  defeat this attack even if the requested block has been pruned.  None
+  of this mitigates the attack for actual SPV clients.
+
+- The [`abandontransaction`][rpc abandontransaction] RPC has been fixed
```

Comment:
nit: inconsistent formating (links to the RPCs above cover the name of the RPC and the word RPC and do not have backticks to render as code)